app/controllers/sessions.rb

```diff
@@ -0,0 +1,29 @@
+
+class Chitter < Sinatra::Base
+
+  get '/sessions/new' do
+    erb :'sessions/new'
+  end
+
+  post '/sessions' do
+    user = User.authenticate(params[:email], params[:password])
+    if user
+      session[:user_id] = user.id
```

Comment:
Can this code be made more declarative of what it does? At the moment, it seems to reveal its implementation, rather than its intention.